components/brave_wallet_ui/panel/container.tsx

```diff
@@ -889,6 +903,27 @@ function Container () {
     )
   }
 
+  if (selectedPanel === 'currencies') {
+    return (
+      <PanelWrapper isLonger={false}>
+        <StyledExtensionWrapper>
+          <Panel
+            navAction={navigateTo}
+            title={panelTitle}
+            useSearch={false}
+          >
+            <ScrollContainer>
+              <SelectCurrency
+                onBack={onBack}
+                onSelectCurrency={onSelectCurrency}
```

Comment:
Could we utilize the `navigateTo` method here for these 2 props? 🙂

`onBack={() => navigateTo('buy')`
`onSelectCurrency={() => navigateTo('buy')`

Reply:
Fixed here: https://github.com/brave/brave-core/commit/c2a4f8d2b7834199a805d767fc829c80cb904aee